packages/@glimmer/global-context/package.json

```diff
@@ -0,0 +1,8 @@
+{
+  "name": "@glimmer/global-context",
+  "version": "0.58.0",
+  "repository": "https://github.com/glimmerjs/glimmer-vm/tree/master/packages/@glimmer/global-context",
```

Comment:
We should update these (in a future PR) to something like:

```json
{
  "repository": {
    "type" : "git",
    "url" : "https://github.com/glimmerjs/glimmer-vm.git",
    "directory": "packages/@glimmer/global-context"
  }
}
```

https://docs.npmjs.com/configuring-npm/package-json.html#repository